Classes/Utility/ExtensionInstallationStatus.php

```diff
@@ -107,8 +107,8 @@ public function getStatusMessage()
             $statusMessage .= '<p>Your Extension is not installed yet.</p>';
             if ($this->usesComposerPath) {
                 $statusMessage .= sprintf(
-                    '<p>Execute <code>composer require %s</code> in terminal',
-                    $this->extension->getComposerInfo()['name']
+                    '<p>Add <code>"%s": "@dev"</code> to the <code>"require"</code> section of your project composer.json<br>Execute <code>composer require %s</code> in terminal',
```

Comment:
use `%1$s` instead of passing the package name twice.

extend the cli command with `composer require %1$s:@dev`